models/classes/websource/BaseWebsource.php

```diff
@@ -145,8 +145,9 @@ public function getMimetype($filePath)
                     }
                     break;
                 case 'svg':
+                case 'svgz':
                     //when there are more than one image in svg file - finfo recognizes it as `image/svg`, while it should be `image/svg+xml` or at least `text/plain` for a previous hack to work
-                    if ($mimeType === 'text/plain' || $mimeType === 'image/svg') {
+                    if ($mimeType === 'text/plain' || $mimeType === 'image/svg' || $mimeType === 'application/x-gzip') {
```

Comment:
I think it reached a point we should have a constant called `ALLOWED_SVGZ_MIMETYPES` and just perform an `in_array` on this. I.e:

```suggestion
                    if (in_array($mimeType, self::ALLOWED_SVGZ_MIMETYPES)) {
```

So if we extend the list, the constant is updated. Not required, but an idea.